src/SOS/SOS.UnitTests/SOS.cs

```diff
@@ -397,35 +397,40 @@ public async Task WebApp3(TestConfiguration config)
             UsePipeSync = true,
             DumpGenerator = SOSRunner.DumpGenerator.DotNetDump
         },
-                Output);
+        Output);
     }
 
     [SkippableTheory, MemberData(nameof(SOSTestHelpers.GetConfigurations), "TestName", "SOS.DualRuntimes", MemberType = typeof(SOSTestHelpers))]
     public async Task DualRuntimes(TestConfiguration config)
     {
-        if (config.PublishSingleFile)
-        {
-            throw new SkipTestException("Single file not supported");
-        }
-        // The assembly path, class and function name of the desktop test code to load/run
-        string desktopTestParameters = TestConfiguration.MakeCanonicalPath(config.GetValue("DesktopTestParameters"));
-        if (string.IsNullOrEmpty(desktopTestParameters))
+        // This test on linux/macOS can be called with an empty config because vstest and dotnet test fail/complain about no test parameters. The
```

Comment:
```suggestion
        // This test on linux/macOS can be called with an empty config because vstest and dotnet test fail/complain if a given test has no inputs for its execution.
```

- A skip exception might also be more appropriate, as this test never ran.